src/accordian/Accordion.tsx

```diff
@@ -0,0 +1,186 @@
+            <div className={ClassNames.ACCORDION_COLLAPSED_CONTENT} aria-hidden="false">
+                <div className={ClassNames.ACCORDION_TOGGLE_CONTENT} style={{display: "block"}}>
+                    <div className={ClassNames.ACCORDION_INNER_CONTENT}>
+                        <div className={ClassNames.ACCORDION_COMPONENT}>{content.component}</div>
+                    </div>
+                </div>
+            </div>
+        );
+    }
+
+    getItemContent = (index: any, title: any, isPrevious: boolean) => {
+        const {accordionMultiPanel} = this.props;
+        let panelClass = isPrevious
+            ? ClassNames.ACCORDION_PANEL_INNER
+            : classNames([ClassNames.ACCORDION_PANEL_INNER, ClassNames.ACCORDION_PANEL_OPEN]);
+        let expanded = isPrevious ? false : true;
+        return (
+            <div
+                role="group"
+                className={panelClass}
+                key={index}
+                onClick={() => this.handleButtonClick(index, accordionMultiPanel)}
+            >
+                <div className={ClassNames.ACCORDION_HEADER}>
+                    <button
+                        className={ClassNames.ACCORDION_HEADER_BUTTON}
+                        type="button"
+                        aria-disabled="false"
+                        aria-controls="clr-accordion-content"
+                        aria-expanded={expanded}
+                    >
+                        <span className={ClassNames.ACCORDION_SR} />
+                        <span className={ClassNames.ACCORDION_STATUS}>
+                            <Icon className={ClassNames.ACCORDION_ANGLE} dir={Direction.RIGHT} shape="angle" />
+                            <span className={ClassNames.ACCORDION_NUMBER}></span>
+                        </span>
+                        <div className={ClassNames.ACCORDION_TITLE}>{title}</div>
+                    </button>
+                </div>
+            </div>
+        );
+    };
+
+    getAccordionContent = () => {
+        const {content, accordionMultiPanel} = this.props;
+        const panelContent = content.map((content, index) => {
+            return {
+                content: (
+                    <div
+                        role="group"
+                        className={ClassNames.ACCORDION_PANEL_INNER}
+                        key={index}
+                        onClick={() => this.handleButtonClick(index, accordionMultiPanel)}
+                    >
+                        <div className={ClassNames.ACCORDION_HEADER}>
+                            <button
+                                className={ClassNames.ACCORDION_HEADER_BUTTON}
+                                type="button"
+                                aria-disabled="false"
+                                aria-controls="clr-accordion-content"
+                                aria-expanded="false"
+                            >
+                                <span className="clr-sr-only" />

```

Comment:
you can use ' ACCORDION_SR'  here which is defined in the ClassNames.

Reply:
addressed